src/bp/nlu-core/predict-pipeline.ts

```diff
@@ -68,31 +68,37 @@ interface E1IntentPred {
 const DEFAULT_CTX = 'global'
 const NONE_INTENT = 'none'
 
+// Gives detected language if found one
+// If language is in bot languages give it to predict
+// Else give default one
```

Comment:
No need for these type of comments, the function code itself should be clear enough

Reply:
My bad it was mine for debug. Removed it